### DISCUSSION TITLE
Time Series Visualization from PostGIS database

### DISCUSSION THREAD

**Discussion by Kakar3737** [opengeos/leafmap#729] [Q&A] [answered]
Hello. Is it possible to make a time series visualization with raster data stored in PostGIS? If yes, how to do it? Thank you for your answer.

**Comment by giswqs** [comment, marked as answer]
If you can generate XYZ tiles from the raster data stored in PostGIS, you can use `Map.add_time_slider()` to visualize time series. See 
https://leafmap.org/notebooks/22_time_slider/
